app/templates/_bower.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "<%= props.license %>",
   "dependencies": { <% for(var i=0; i<components.length; i++) { %>
-    "<%= components[i].name %>": "<%= components[i].ver%>"<% if (i < components.length - 1) { %>,<% } %><% } %>
+    "<%= components[i].name %>": "<%= components[i].ver%>"<% if (i < components.length - 1) { %>,<% } %><% } %>,
+    "jquery.easing": "~1.3.0"
```

Comment:
Probably for `https://github.com/jimmynotjim/generator-cf/blob/gulp-option/app/templates/gulp/config/browserify-shims.js`